2_create_app_namespace.sh

```diff
@@ -3,27 +3,25 @@ set -euo pipefail
 
 . utils.sh
 
-announce "Creating Test App namespace."
+announce "Creating Application namespace."
 
 if [[ $PLATFORM == openshift ]]; then
   $cli_with_timeout "login -u $OPENSHIFT_USERNAME"
 fi
 
-set_namespace default
```

Comment:
don't we need this?

Reply:
No because we don't want to deploy anything to a default namespace, only app namespace or conjur namespace. Here, we are just creating a namespace and switching over to it so I didn't see any value in setting the namespace to default